### DISCUSSION TITLE
Port for Nova

### DISCUSSION THREAD

**Discussion by dklaver15** [daltonmenezes/aura-theme#145] [Port request]
I can work on a port for Nova.

**Comment by daltonmenezes** [comment]
@DanKlaver15 awesome, I'm going to assign you to #143 , thanks 💜

**Reply by daltonmenezes** [reply]
@DanKlaver15 just let me know If you need any help, also don't forget to read the [Documents](https://github.com/daltonmenezes/aura-theme#documentation). 🚀
